js/binance.js

```diff
@@ -7011,6 +7012,96 @@ module.exports = class binance extends Exchange {
         return await this[method] (this.extend (request, params));
     }
 
+    async fetchSettlementHistory (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        /**
+         * @method
+         * @name binance#fetchSettlementHistory
+         * @description fetches historical settlement records
+         * @see https://binance-docs.github.io/apidocs/voptions/en/#historical-exercise-records
+         * @param {string} symbol unified market symbol of the settlement history
+         * @param {int} since timestamp in ms
+         * @param {int} limit number of records, default 100, max 100
+         * @param {object} params exchange specific params
+         * @returns {[object]} a list of [settlement history objects]
+         */
+        await this.loadMarkets ();
```

Comment:
@Dan-krm doesn't Binance support this endpoint for swaps? If not, we should add that check

Reply:
I think this method is mainly for options and delivery futures, I haven't noticed any endpoints for futures, but I'm thinking we'll eventually create a new method called fetchMySettlementHistory and I think there is a futures endpoint for that. I will add a check to this method